lib/libzpool/Makefile.am

```diff
@@ -154,7 +159,24 @@ KERNEL_C = \
 	zio_inject.c \
 	zle.c \
 	zrlock.c \
-	zthr.c
+	zstd.c \
+	zthr.c \
+	zstd/freebsd/zstd_kmalloc.c \
+	zstd/common/zstd_common.c \
+	zstd/common/fse_decompress.c \
+	zstd/common/entropy_common.c \
+	zstd/common/xxhash.c \
+	zstd/compress/hist.c \
+	zstd/compress/zstd_compress.c \
+	zstd/compress/fse_compress.c \
+	zstd/compress/huf_compress.c \
+	zstd/compress/zstd_double_fast.c \
+	zstd/compress/zstd_fast.c \
+	zstd/compress/zstd_lazy.c \
+	zstd/compress/zstd_ldm.c \
+	zstd/compress/zstd_opt.c \
+	zstd/decompress/zstd_decompress.c \
+	zstd/decompress/huf_decompress.c
```

Comment:
but lua is also a different kernel module. i suggest not to model it in the same way because the kernel has a own zstd module implementation. this will create name conflicts